### PR TITLE
Add comic emoticons by Ravie

### PR DESCRIPTION
**Pull request by def-**
@HiRavie Fine by you?

Screenshot 2023-02-28 at 19 50 22
## Checklist

- [x] Tested the change ingame
- [x] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/assets/entities/license.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 comfort:
 Copyright louis
-CC BY-SA 3.0 license (http://creativecommons.org/licenses/by-sa/3.0/)
+CC BY-SA 3.0 license (httpss://creativecommons.org/licenses/by-sa/3.0/)
```
Comment (Chairn) [inline data/assets/entities/license.txt:3]: https

**Comment by HiRavie** [comment]
> @HiRavie Fine by you?

Of course, that's awesome

**Comment by Chairn** [comment]
bors r+
Thanks for the awesome emotes :wink:

**Comment by heinrich5991** [comment]
This has a license that is not considered an open-source license. We went out of our way to get rid of them for skins a while back. I suggest we remove this again or ask whether @HiRavie would consider an open-source license.

Both ND and NC are a problem for distributions like Debian: https://wiki.debian.org/DFSGLicenses, https://wiki.debian.org/DebianFreeSoftwareGuidelines.

**Comment by def-** [comment]
Oh well, too bad. I wasn't sure if this license would cause problems and isn't considered open source. @HiRavie remove or relicence?

**Comment by HiRavie** [comment]
I'd rather not change the license, it's fine if people have to download from forum.

**Pull request merged**